pdc_client/plugins/build_image_rtt_tests.py

```diff
@@ -55,13 +55,11 @@ def list_build_image_rrt_tests(self, args):
         if build_images_rrt:
             print_build_image_rtt_list(build_images_rrt)
 
-    def build_image_rrt_tests_info(self, args, rtt_id=None):
-        if not rtt_id:
-            rtt_id = self._get_build_image_rtt_id(args.build_nvr, args.image_format)
+    def build_image_rrt_tests_info(self, args):
+        rtt_id = self._get_build_image_rtt_id(args.build_nvr, args.image_format)
```

Comment:
After api url already support build_nvr and image_format, why we still need this. Could we just use line 62 directly?